.build/package.json

```diff
@@ -4,7 +4,7 @@
   "version": "0.0.1",
   "description": "Guides for Red Hat OpenShift Streams for Apache Kafka",
   "port": "9001",
-  "repository": "https://github.com/bf2fc6cc711aee1a0c2a/mas-guides.git",
+  "repository": "https://github.com/redhat-developer/app-services-guides.git",
   "homepage": "https://cloud.redhat.com/application-services/openshift-streams",
   "license": "MIT",
```

Comment:
I should probably change this to Apache 2.0 while I am here?

Reply:
No problem. Cloud mosaic is MIT